src/components/atoms/ToLoginMessage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {View, Text, Pressable} from 'react-native';
+import style from '../../styles/ToLoginMessage';
+
+export const ToLoginMessage = () => {
+  const goToLoginScreen = () => {
+    console.log('to login screen');
```

Comment:
Do not upload console.log() code to GitHub, if you want to display a message I recommend using the [Alert](https://reactnative.dev/docs/alert) component.